apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.ValueConversion.cs

```diff
@@ -162,6 +162,12 @@ internal Value ToProtobufValue(object value, SpannerConversionOptions options)
                         throw new ArgumentException("SpannerStruct cannot be used for string parameters", nameof(value));
                     }
                     return new Value { StringValue = Convert.ToString(value, InvariantCulture) };
+                case TypeCode.Json:
+                    if (value is string jsonString)
+                    {
+                        return new Value { StringValue = jsonString };
```

Comment:
Note to self: after this is merged, I'll rewrite all of these "new Value" to "Value.FromString" etc.